style.css

```diff
@@ -40,18 +46,47 @@ header {
   background-size: cover;
   background-position: center;
   margin-bottom: 120px;
+  transition: all 0.4s;
 }
 
 #nav {
   display: flex;
   justify-content: space-between;
+  align-content: center;
   align-items: center;
   padding: 0 24px;
   width: 100%;
 }
 
-#nav-links {
+@keyframes fade-open {
+  from {
+    transform: translateY(-800px);
+    opacity: 0;
+  }
+
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+.nav-links {
   display: none;
+  width: 100%;
+  border-radius: 0 0 30px 30px;
+}
+
+.nav-links.open {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 90vh;
+  background-color: #fff;
+  margin-top: 0;
+  padding: 20px;
+  animation: fade-open 0.5s;
 }
```

Comment:
[OPTIONAL] You did a great job with your mobile menu, well done 😄! I liked the animation effect as well. Please take into account that most of the mobile menus stick to the user's screen. This means that, when scrolling down, the menu is still displayed at the top of the page. Currently, your menu is static at the beginning of your page, causing this effect when scrolling down: 
![image](https://user-images.githubusercontent.com/42799579/201776600-b312952b-143e-4a8e-bca7-9d11455c0cd6.png) 

So, to improve the UX/UI of your webpage, kindly consider this advice 😄. 
> You could do it by giving a position: fixed to your menu 😉.